snuba/datasets/storages/errors.py

```diff
@@ -63,6 +79,7 @@
         ("trace_id", Nullable(UUID())),
         ("partition", UInt(16)),
         ("offset", WithCodecs(UInt(64), ["DoubleDelta", "LZ4"])),
+        ("message_timestamp", DateTime()),
```

Comment:
This _might_ compress decently well with delta compression (as long as external and internal latency isn't _too_ wide of a distribution.)